docs/blocks.py

```diff
@@ -0,0 +1,168 @@
+import textwrap
+import xml.etree.ElementTree as etree
+from typing import Any
+
+from pymdownx.blocks import BlocksExtension  # pyright: ignore[reportMissingTypeStubs]
+from pymdownx.blocks.block import (  # pyright: ignore[reportMissingTypeStubs]
```

Comment:
I decided to do a drive-by linting of their project as a meditative activity: https://github.com/facelessuser/pymdown-extensions/pull/2634